test/create_module_emr-iam_test.go

```diff
@@ -0,0 +1,33 @@
+package test
```

Comment:
I'm assuming having the terratest test was helpful in your development, but including it here is slightly out of scope for this epic.

I'm fine with keeping it around though, this test is benign enough even if we haven't setup the integrations yet or refactor the format later it won't be a large change.

Reply:
Yeah, I agree this would make more sense only if we had other tests merged already. I'm afraid though that the check from `continuous-integration/jenkins/pr-merge` won't merge because of *.go files check failing. Please look into the jenkins job and If that's the case let me know... so that we merge using the fastest route (maybe leave these out for now)